### PR TITLE
Trigger redraw on WM_SHOWWINDOW event

### PR DESCRIPTION
**Pull request by knopp**
Add ability to trigger redraw by embedding app sending WM_SHOWWINDOW with wParam TRUE 

https://github.com/flutter/flutter/issues/75319

I'm not entirely sure repurposing WM_SHOWWINDOW is the best approach here, it could be a custom message as well.

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [X] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on
writing and running engine tests.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [X] I signed the [CLA].
- [X] All existing and new tests are passing.
- [ ] The reviewer has submitted any presubmit flakes in this PR using the [engine presubmit flakes form] before re-triggering the failure.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/master/CONTRIBUTING.md#style
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[engine presubmit flakes form]: https://forms.gle/Wc1VyFRYJjQTH6w5A

### CODE REVIEW

**Comment by clarkezone** [comment]
As this PR stands, it breaks UWP compilation mode.  Please add the corresponding implementation for `FlutterWindowWinUWP` and ensure that .the solution builds correctly when configured with the --winrt flag

**Comment by knopp** [comment]
Sorry about it. I did try the UWP build but it fails with unrelated `error: no member named 'wait_for' in namespace 'winrt::impl'` so I need to deal with that first.

**Comment by knopp** [comment]
I think the problem with UWP build was caused by WindowBindingHandlerDelegate ::OnShow() not being pure virtual. Fixed [here](https://github.com/flutter/engine/pull/24164/commits/54088afbda70866be1cdd862af930d3814baeb70).

File: shell/platform/windows/flutter_windows_view.cc [hunk 1]
```diff
@@ -119,6 +119,14 @@ void FlutterWindowsView::OnWindowSizeChanged(size_t width, size_t height) {
   }
 }
 
+void FlutterWindowsView::OnShow() {
+  if (resize_status_ == ResizeState::kDone) {
+    // Request new frame
```
Comment (stuartmorgan) [inline shell/platform/windows/flutter_windows_view.cc:124]: Please reference https://github.com/flutter/flutter/issues/69716 here.

**Review by stuartmorgan — COMMENTED** [review]
Am I understanding correctly that this require the runner to manually forward this event? If so, a specific Windows embedding API would be better; forwarding messages leads to much harder-to-understand codepaths. (We used to do that for system font changes, and it was entirely non-obvious why the non-top-level window code had a handler for a message that only top level windows get).

**Comment by knopp** [comment]
Indeed, this relies on embedder forwarding the WM_SHOWWINDOW message. If you think a method on FlutterDesktopViewController is better way to address it I'll submit another PR.

**Pull request closed without merging**